cmake/llvm-project.cmake

```diff
@@ -43,26 +42,26 @@ list(APPEND CMAKE_MODULE_PATH
   "${ROCM_PATH}/hip/cmake"
 )
 list(APPEND CMAKE_MODULE_PATH
-  "${CMAKE_CURRENT_BINARY_DIR}/lib/cmake/mlir"
+  "${CMAKE_BINARY_DIR}/lib/cmake/mlir"
```

Comment:
`LLVM_BINARY_DIR`, perhaps? Or is there something else here?

Reply:
Ok, hold on, what's the problem upstream? `llvm-project.cmake` here is ours, not theirs.